internal/namespaces/documentdb/v1beta1/custom.go

```diff
@@ -0,0 +1,36 @@
+package documentdb
+
+import (
+	"github.com/fatih/color"
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+	"github.com/scaleway/scaleway-cli/v2/internal/human"
+	documentdb "github.com/scaleway/scaleway-sdk-go/api/documentdb/v1beta1"
+)
+
+var (
+	instanceStatusMarshalSpecs = human.EnumMarshalSpecs{
+		documentdb.InstanceStatusUnknown:      &human.EnumMarshalSpec{Attribute: color.Faint, Value: "unknown"},
+		documentdb.InstanceStatusReady:        &human.EnumMarshalSpec{Attribute: color.FgGreen, Value: "ready"},
+		documentdb.InstanceStatusProvisioning: &human.EnumMarshalSpec{Attribute: color.FgBlue, Value: "provisioning"},
+		documentdb.InstanceStatusConfiguring:  &human.EnumMarshalSpec{Attribute: color.FgBlue, Value: "configuring"},
+		documentdb.InstanceStatusDeleting:     &human.EnumMarshalSpec{Attribute: color.FgBlue, Value: "deleting"},
+		documentdb.InstanceStatusError:        &human.EnumMarshalSpec{Attribute: color.FgRed, Value: "error"},
+		documentdb.InstanceStatusAutohealing:  &human.EnumMarshalSpec{Attribute: color.FgBlue, Value: "auto-healing"},
+		documentdb.InstanceStatusLocked:       &human.EnumMarshalSpec{Attribute: color.FgRed, Value: "locked"},
+		documentdb.InstanceStatusInitializing: &human.EnumMarshalSpec{Attribute: color.FgBlue, Value: "initialized"},
+		documentdb.InstanceStatusDiskFull:     &human.EnumMarshalSpec{Attribute: color.FgRed, Value: "disk_full"},
+		documentdb.InstanceStatusBackuping:    &human.EnumMarshalSpec{Attribute: color.FgBlue, Value: "backuping"},
+		documentdb.InstanceStatusSnapshotting: &human.EnumMarshalSpec{Attribute: color.FgBlue},
+		documentdb.InstanceStatusRestarting:   &human.EnumMarshalSpec{Attribute: color.FgBlue},
```

Comment:
Why the value is omitted here?

Reply:
Values are not required, if not given it will use the value of the enum.
The values are given for others as I copied it from rdb